### ISSUE TITLE
ROS Adjustment factors

### ISSUE THREAD

**Issue by aklibenali** [firelab/farsite#5]
How do I define in the simulation inputs the Rate-of-spread adjustment factor for each fuel model?

**Comment by aklibenali** [comment]
Based on the reply to the other question, and just for the record, the switch is: 

ROS_ADJUST_FILE: /home/latwood/Documents/windninaPanther/test1177973/currentInput/test.adj

**Comment by latwood** [comment]
Hi aklibenali,

As I said on the issue with gridded winds, if it's not in the examples, you'll probably have to refer to the test1177973.input file and maybe even try to look directly at the code. If you do end up looking at the code, be careful to ignore any unused code.

It looks like none of the examples have this, but it does look like I saw some kind of optional method for this in the code when I was looking at the code directly a couple years ago. https://github.com/firelab/farsite/blob/master/examples/test1177973.input#L122. Hm, looks like when I compiled that list of comments, I had some test case files that never found their way into the repo, probably lost them. So it looks like I may have tried it out, not sure if it actually works. You'll probably have try to reconstruct that file.

Loren

**Comment by latwood** [comment]
Looks like you added that info faster than I was able to. Yup that's the line you would need to add such a file, but I seem to have not added that file to the test cases. I'll do a quick check to see if I still have any such file, but I'm pretty sure I pulled what I thought was unique from my personal test cases to make the repo test cases and got rid of the rest. I didn't realize that I had accidentally lost that file till you brought this up.

**Comment by aklibenali** [comment]
Well, from a user's perspective it is quite easy to create such file, so it shouldn't be a problem.

Thanks for your help.

**Comment by latwood** [comment]
Yeah it looks like I don't have this file anymore. Hopefully what was found in that test1177973.input file helps in making the file. Let me know if you have any trouble with this.
